code/game/gamemodes/malfunction/malfunction.dm

```diff
@@ -257,7 +257,7 @@
 		world << "<B>The self-destruction of [station_name()] killed everyone on board!</B>"
 
 	else if ( station_captured &&  malf_dead && !station_was_nuked)
-		feedback_set_details("round_end_result","halfwin - AI killed, staff lost control")
+		feedback_set_details("round_end_result","draw - AI killed, staff lost control")
 		world << "<FONT size = 3><B>Neutral Victory</B></FONT>"

```

Comment:
This should either be a minor human victory (purged the malf AI, but NT has to spend money on fixing APCs) or a major AI victory (taking control allowed the AI to install failsafes in the event of its death. Station self-destructing in 10...). A draw here makes little sense.